product_docs/docs/pem/9/certificates/regenerating_agent_certificates.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: "Regenerating the agent self-signed SSL certificates"
```

Comment:
It might be useful to add something on this page to explain which of the steps are still relevant if you are using externally provided certificates

Reply:
No steps are relevant when external certificates are used

Reply:
Surely if the external certs expire then they must be replaced, so at a minimum the 'stop the agent, copy the files, restart the agent' steps are valid. Perhaps just add a note to that effect at the start i.e. "If you are using external certificates, simply stop the agent service, replace the certs and restart"

Reply:
I spoke with @asheshv about this and just a note on appropriate bit of the the index.html page saying 'repeat these steps to renew your external certs when they expire' should be sufficient.